wgsl/index.bs

```diff
@@ -4958,10 +4958,6 @@ An override-expression |E| [=behavioral requirement|will=] be evaluated if and o
     [=behavioral requirement|will=] be evaluated, and evaluation of |OuterE|
     requires |E| to be evaluated.
 
-Note: An override-expression may not be usable as the initializer for an
```

Comment:
The following is an override expression, but can't be used to initialize an override declaration:
```
override a : i32;
override b = vec2(a,a); // invalid, b must be a scalar type
```

Reply:
I have clarified the note instead of removing it. Let me know if it looks good!